### DISCUSSION TITLE
Vercel Postgres extension compatibility

### DISCUSSION THREAD

**Discussion by TabithaLarkin** [vercel/community#4450] [Feature Question] [answered]
### Question

The [supported extensions list](https://vercel.com/docs/storage/vercel-postgres/supported-postgresql-extensions) in the Vercel documentation differs from what is available on the [Neon website](https://neon.tech/docs/extensions/pg-extensions). Vercel seems to support up to `postgis_tiger_geocoder`, but the Neon list continues on past this. From what I can tell the features are largely mirrored, but the vercel list seems to end early.

Does this mean that Vercel does not yet support these features, or is it an error in the documentation?

**Comment by swarnava** [comment, marked as answer]
Hi there,

It looks like our doc is outdated and we will fix this ASAP. Neon doc is up to date :)
